hdf5-types/src/string.rs

```diff
@@ -182,6 +182,38 @@ impl_string_traits!(VarLenUnicode, VarLenUnicode);
 
 // ================================================================================
 
+// use HDF5 malloc/free for variable length objects on Windows
+// with a sufficient HDF5 version
+#[cfg(windows)]
+mod alloc {
+    use hdf5_sys::h5::herr_t;
+    use libc::{c_void, size_t};
+
+    pub unsafe fn malloc(size: size_t) -> *mut c_void {
+        hdf5_sys::h5::H5allocate_memory(size, 0)
+    }
+
+    pub unsafe fn free(mem: *mut c_void) -> herr_t {
+        hdf5_sys::h5::H5free_memory(mem)
+    }
+}
+
+// use libc for variable length objects outside of windows
+#[cfg(not(windows))]
+mod alloc {
+    use hdf5_sys::h5::herr_t;
+    use libc::{c_void, size_t};
+
+    pub unsafe fn malloc(size: size_t) -> *mut c_void {
+        libc::malloc(size)
+    }
+
+    pub unsafe fn free(mem: *mut c_void) -> herr_t {
+        libc::free(mem);
+        0
+    }
+}
+
```

Comment:
This is fixed in the `dcpl` branch. You can ignore this and the test failures for windows DLL

Reply:
removed.